src/ui/ui-kit/List/ListItem.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import cn from 'classnames';
+import type { LinkProps } from 'react-router-dom';
+import { UnstyledLink } from '../UnstyledLink';
+import { UnstyledAnchor } from '../UnstyledAnchor';
+import { UnstyledButton } from '../UnstyledButton';
+import * as styles from './styles.module.css';
+
+export const ListItemLink = React.forwardRef(
+  (
+    { className, style, ...props }: LinkProps,
+    ref: React.Ref<HTMLAnchorElement>
+  ) => {
+    return (
+      <UnstyledLink
+        ref={ref}
+        style={{ color: 'inherit', ...style }}
+        className={cn(styles.item, 'parent-hover', className)}
```

Comment:
How is `parent-hover` class used?

Reply:
This allows us to style inner elements' hover without additional props

Reply:
No, now it is not used. Just a helper for the future